.mergify.yml

```diff
@@ -25,5 +27,7 @@ pull_request_rules:
       - "#changes-requested-reviews-by=0"
       - "#review-threads-unresolved=0"
       - -approved-reviews-by~=author
-      - check-success=Quality Gate
       - check-success=Validate PR title
+      - or:
+        - check-success=Quality Gate
+        - -files~=^(?!docs/)
```

Comment:
beautiful negative lookahead

Could you add the following things as well?

- All the root-level .md files
- logo/